about.html

```diff
@@ -0,0 +1,166 @@
+
+  <link rel="stylesheet" href="./style.css">
+  <!-- COCOGOOSE & LATO FONT -->
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/lato" rel="stylesheet">
+  <!-- Link to FontAwesome page -->
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css" 
+  integrity="sha384-oS3vJWv+0UjzBfQzYUhtDYW+Pj2yciDJxpsK1OYPAYjqT085Qq/1cq5FLXAZQ7Ay"
+  crossorigin="anonymous"/>
+
+</head>
+<body>
+  <header>
+    <nav class="nav-bar">
+      <section class="nav-bar-section1">
+        <div><i onclick='openWindow("erik-fb")' class="fab fa-facebook-f"></i></div>
+        <div><i onclick='openWindow("lore-ig")' class="fab fa-instagram"></i></div>
+        <div>English</div>
+        <div>My Page</div>
+        <div>Logout</div>
+      </section>
+      
+      <section class="nav-bar-section2">
+        <div class="nav-bar-logo">
+         <a href="./index.html"><img src="assets/Logo1.png" alt=""></a> 
+        </div>
+        <ul class="nav-bar-options">
+          <li><a href="./about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li><a href="#">News</a></li>
+        </ul>
+        <div class="nav-bar-button">IG Campaign</div>
+      </section>
+
+      <!-- Hamburguer icon -->
+      <section class="hamburguer-button">
+          <i id="burguer-icon" class="fas fa-bars"></i>
+      </section>
+
+    </nav>
+  </header>
+
+  <main class="about-main">
+    <section class="intro-containerAbout">
+      
+      <section class="about-navbar">
+        <i class="fas fa-igloo"></i>
+        <i class="fas fa-chevron-right"></i>
+        <span>About</span>
+        <i class="fas fa-chevron-right"></i>
+        <span>IG Campaign</span>
+      </section>
+
+      <div class="intro-part1">
+        
+        <h1>INTERCULTURAL EXCHANGE <br> MEXICO-CANADA 2023</h1>
+        <hr>
+        <p class="intro-paragraph1">The International Exchange brings the opportunity to mingle our way of perceiving the collision of the world as each day the internet becomes an intrinsic part of it. This event is hosted one year in Mexico with Canadian speakers, the next in Canada with Mexican speakers and then the pattern repeats itself.
+          In this memorable occasion we wish to provide everyone with contacts that may last a lifetime and give support to all your endeavors, by providing open knowledge along with embracing everyone’s culture.
+          Then if you feel excited to entwine your path with ours, let us support each other by sharing all the easy tools the present has to offer.</p>
+        <p class="intro-paragraph2">Please contact us per Email for any further questions about IE Mexico-Canada 2023!</p>
+        <p class="intro-paragraph3">eriksanchezbeltran@outlook.com <span><i class="fas fa-chevron-right"></i></span></p>
```

Comment:
- [ ] You have done a great job styling your email. However, it would be semantically correct and a user experience improvement to make it a link. Below is how you can do it. 
```<p><a href="mailto:eriksanchezbeltran@outlook.com">eriksanchezbeltran@outlook.com</a></p>``` . 

This way your users will get automatically redirected to their email application to directly mail you. Kindly make this small change.

Reply:
**Excellent recommendation! I didn't know how to do this.**

I have already applied it to both the mobile and desktop versions

About page - This is what it looks like in the browser
-
![image](https://user-images.githubusercontent.com/106561762/181601247-aa5790d4-0800-4137-8755-a58740e544e7.png)

this is what it looks like inside the mail app
-
![image](https://user-images.githubusercontent.com/106561762/181602875-48c9571a-f8d7-4c7a-b739-db82b6436dc4.png)

Final code
-
![image](https://user-images.githubusercontent.com/106561762/181603012-f3bac965-b8a2-4cef-a690-d7f1dde70e61.png)

-Thank you so much for your advice! I am sure I will use this knowledge again.

Reply:
Great job @ErikStoupignan  you actually took it further and expanded on what I had provided. This is growth now :clap: